static/bidder-params/emx_digital.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Should this file be renamed as well to reflect cadent_aperture_mx? Similarly, the `static/bidder-info/emx_digital.yaml` file

Reply:
We were under the impression that this file needs to stay so that we are able to alias the `cadent_aperture_mx` bidder using `emx_digital`.  We've created a new `static/bidder-params/cadent_aperture_mx.json` file, and when we created our previous PR, we followed the steps on the [FAQ for aliasing an adapter,](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#aliasing-an-adapter) and it seemed like we needed to keep the bidder-params files for both bidder names we plan to use, the alias and the alias target.  Please advise if this is incorrect information?

Reply:
Your understanding is correct. These files do not need to be changed.